AWS-GD datetime issue fix (#12245)
demisto/content: Packs/AWS-GuardDuty/ReleaseNotes/1_1_1.md

#### Integrations
##### AWS - GuardDuty

> - Fixed an issue where the ***aws-gd-get-findings*** command was throwing a datetime error.

Reviewer: ```suggestion
- Fixed an issue where the ***aws-gd-get-findings*** command failed to execute.
```

Reviewer: better to avoid pythonic explanations.
